client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -848,47 +784,19 @@ export class PlanFeatures2023Grid extends Component<
 				! isWooExpressPlusPlan( properties.planName )
 		);
 
-		return planProperties
```

Comment:
This block is moved to client/my-sites/plan-features-2023-grid/components/plan-features.tsx above.